main/src/ca/uwaterloo/flix/language/phase/Parser.scala

```diff
@@ -148,7 +148,7 @@ class Parser(val source: Source) extends org.parboiled2.Parser {
         }
 
         def CaseWithType: Rule1[ParsedAst.Case] = namedRule("Case") {
-          SP ~ Names.Tag ~ Type ~ SP ~> ParsedAst.Case
+          SP ~ Names.Tag ~ Types.Tuple ~ SP ~> ParsedAst.Case
```

Comment:
Does this still allow a record? I guess it does?

Reply:
You can do a record row like `C(())` but that would be ill-kinded anyway.

A proper record `C({})` is still possible.